app/Agri.Data/Agri.Data.csproj

```
@@ -8,6 +8,7 @@
     <None Remove="SeedData\DepthsData.json" />
     <None Remove="SeedData\StaticDataVersion_10.json" />
     <None Remove="SeedData\StaticDataVersion_11.json" />
+    <None Remove="SeedData\StaticDataVersion_13.json" />
```

```suggestion
    <None Remove="SeedData\StaticDataVersion_12.json" />
```
In https://github.com/bcgov/agri-nmp/pull/635/, we set the remove to the previous version. Still not totally sure what is necessary, but it DID work in that PR